integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/MetalakePageTest.java

```diff
@@ -113,4 +113,53 @@ public void testLinkToCatalogsPage() throws InterruptedException {
     metalakePage.clickMetalakeLink(name);
     Assertions.assertTrue(metalakePage.verifyLinkToCatalogsPage(name));
   }
+
+  @Test
+  @Order(9)
+  public void testRefreshPage() {
+    driver.navigate().refresh();
+
+    Assertions.assertEquals(driver.getTitle(), "Gravitino");
+    Assertions.assertTrue(metalakePage.verifyRefreshPage());
+  }
+
+  @Test
+  @Order(10)
+  public void testCheckLinkDatastrato() {
+    String originalWindowHandle = driver.getWindowHandle();
+    metalakePage.footerLinkDatastrato.click();
+    String expectedUrl = "https://datastrato.ai/";
+    Assertions.assertTrue(
+        metalakePage.verifyLinkInNewWindow(originalWindowHandle, expectedUrl, false));
+  }
+
+  @Test
+  @Order(11)
+  public void testCheckLinkLicense() {
+    String originalWindowHandle = driver.getWindowHandle();
+    metalakePage.footerLinkLicense.click();
+    String expectedUrl = "https://github.com/datastrato/gravitino/blob/main/LICENSE";
+    Assertions.assertTrue(
+        metalakePage.verifyLinkInNewWindow(originalWindowHandle, expectedUrl, false));
+  }
+
+  @Test
+  @Order(12)
+  public void testCheckLinkDocs() {
+    String originalWindowHandle = driver.getWindowHandle();
+    metalakePage.footerLinkDocs.click();
+    String expectedUrl = "https://datastrato.ai/docs/";
```

Comment:
Please define static final variable in the class head and replace this, maybe like this
```
private static final String FOOTER_LINKE_DOCS = "https://datastrato.ai/docs/";
```

and another places.